### ISSUE TITLE
The bot is suddenly slow?

### ISSUE THREAD

**Issue by jinxoffender** [finndev/PokeBuddy#783]
About 6 hours ago the bot was catching up to 4 pokemon in a minute now its caught 5 in 7 minutes. the only thing ive done is stopped the bot and started it again. is there a limit setting that can be changed?.

**Comment by LordNoxxious** [comment]
Yes, I also noticed the bot is very slow today. Slow at updating pokestops, slow at starting to move after stopping at pokestop and the ui is much slower at updating information/map.

**Comment by jinxoffender** [comment]
the xp per hour on mine dropped massively as i was on 50k/ hour now its 16k/ hour apart from that the bot works fine

**Comment by jonson09** [comment]
same here, working stable, updatetime for next pokestop is now higher. maybe this is a consequence of a lot of request, or a action to protect us users, or for a better stability, i don´t know but gladful that it is  running now. thx!

**Comment by LordNoxxious** [comment]
Just redownloaded, still having same issues.

**Comment by Thorben24** [comment]
Same for me, i could easily solve it by close the bot and open it new. Before you press start disable all tasks without Farm Pokestops. Just check the first one is enabled, the others disabled. After it was searching more than 2 pokestops you can add all the other task features.

**Comment by Kokainium** [comment]
still slow -_-

**Comment by curtiola** [comment]
Slow ain't so bad though, I prefer a steady 10k an hour, level up too fast and you may be flagged, level 30 or so in just a few days after account creation is not possible for a human and probably stands out

**Comment by Kokainium** [comment]
true true :) but the walkspeed per hour doesnt matter anymore mine is at 35 but walk like 10 :) but its fine :D

**Comment by curtiola** [comment]
I keep mine at 11 usually, but lately I've been trying 3, keep it more realistic, plus he doesn't move so far from the start point that way

**Comment by Kokainium** [comment]
well thats a good point :)

**Comment by Kokainium** [comment]
250k xp just today not bad :P

**Comment by LordNoxxious** [comment]
I keep mine at 15. it moves fast enough for good xp/h and egg hatching.
